tasks/ostree-image-compose

```diff
@@ -10,6 +10,7 @@ base_dir="$(dirname $0)/.."
 HOMEDIR=$(pwd)
 rm -rf ${HOMEDIR}/output
 mkdir -p ${HOMEDIR}/output
+ln -fs ${HOMEDIR}/output ${HOMEDIR}/logs
```

Comment:
Actually probably should have made this ln -fs ${HOMEDIR}/output/logs ${HOMEDIR}/logs